WordPress/Classes/ViewRelated/Notifications/Controllers/NotificationDetailsViewController.swift

```diff
@@ -98,6 +98,10 @@ class NotificationDetailsViewController: UIViewController {
         return DefaultContentCoordinator(controller: self, context: mainContext)
     }()
 
+    lazy var router: NotificationContentRouter = {
```

Comment:
Do you think it'd make sense, for future testability, to abstract the router behind a protocol? (I assume this is not private do that it can be set by a client (i.e. a test). If that's not the case, how would you feel about making it private?

Reply:
These Routers are quite specialized, this one for example has more public methods than the `ActivityContentRouter` but both of them use a generic `ContentCoordinator`, so I think the best way of testing them is indirectly via replacing its coordinator and checking what it tries to present.

Does this make sense?
But still, to test the controller this way, we would need to inject it somehow. I will check this out on the next PR. 👍